lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -50,10 +50,20 @@ function dosomething_metatag_preprocess_page(&$vars) {
   // Add relevant image metatags.
   dosomething_metatag_add_metatag_image($node);
 
+  // Add explicit facebook og meta tags to campaign pages.
+  if (isset($node) && $node->type == 'campaign') {
+    $facebook_data = array(
+      'type' => 'dosomething_facebook:campaign',
+      'url' => url(current_path(), array('absolute'=> TRUE)),
+      'title' => $node->title,
+      'description' => "Volunteer and take action! " . $node->field_call_to_action[$node->language][0]['value'],
```

Comment:
does this work on different translations? 
check [this](https://github.com/DoSomething/phoenix/blob/872486f8f1528224cab1db71d93db732f0a6e5cb/lib/modules/dosomething/dosomething_global/dosomething_global.module#L496)